pybamm/models/submodels/thermal/isothermal/isothermal.py

```diff
@@ -33,6 +35,10 @@ def get_fundamental_variables(self):
         T_cp = T_x_av
 
         variables = self._get_standard_fundamental_variables(T, T_cn, T_cp)
+        variables.update(
```

Comment:
could this be moved to `base_thermal`to avoid repeating in each of the separate base classes?